integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -86,6 +86,10 @@
            <class name="org.wso2.choreo.connect.tests.testcases.withapim.websocket.throttle.WebsocketApiLevelThrottleTestCase"/>
            <class name="org.wso2.choreo.connect.tests.testcases.withapim.websocket.throttle.WebsocketApplicationThrottleTestCase"/>
            <class name="org.wso2.choreo.connect.tests.testcases.withapim.websocket.throttle.WebsocketSubscriptionThrottleTestCase"/>
+           <class name="org.wso2.choreo.connect.tests.testcases.withapim.Http2ClearTextApiTestCase"/>
+           <!-- TODO: Amila Find a way to add backend cert to apim project -->
```

Comment:
https://github.com/wso2/product-microgateway/blob/c5df244bb978152aaba3f3c8f4dbed2611308ec5/integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/standalone/CcWithDefaultConf.java#L70

Reply:
One you have suggested is for apictl project. I wanted the same in apim project. Could do it like below,
```
publisherRestClient.uploadEndpointCertificate(certFile, "http2-secured-endpoint", "https://mockBackend:2351");
```